eva-server/src/main/java/uk/ac/ebi/eva/server/ws/RegionWSServer.java

```diff
@@ -114,4 +114,21 @@ public QueryResponse getVariantsByRegion(@PathVariable("regionId") String region
     public QueryResponse getVariantsByRegion() {
         return setQueryResponse("");
     }
+
+    @RequestMapping(value = "", method = RequestMethod.GET)
```

Comment:
@cyenyxe , Is the "segments" path clear enough, or should we add "/chromosomes"?